### PR TITLE
Move testcontainers to 'test' scope

### PR DESCRIPTION
**Pull request by alebastrov**

### CODE REVIEW

File: oap-hadoop/pom.xml [hunk 1]
```diff
@@ -63,8 +63,22 @@
...
+            <groupId>com.github.docker-java</groupId>
+            <artifactId>docker-java-core</artifactId>
+            <version>3.3.4</version>
+            <scope>test</scope>
+        </dependency>
+        <!-- https://mvnrepository.com/artifact/com.github.docker-java/docker-java-transport-zerodep -->
+        <dependency>
+            <groupId>com.github.docker-java</groupId>
+            <artifactId>docker-java-transport-zerodep</artifactId>
+            <version>3.3.4</version>
```
Comment (JuliaGalabut) [inline oap-hadoop/pom.xml:79]: can you, please, move it to property

File: oap-hadoop/pom.xml [hunk 2]
```diff
@@ -63,8 +63,22 @@
                     <artifactId>slf4j-api</artifactId>
                 </exclusion>
             </exclusions>
+            <scope>test</scope>
         </dependency>
 
+        <dependency>
+            <groupId>com.github.docker-java</groupId>
+            <artifactId>docker-java-core</artifactId>
+            <version>3.3.4</version>
```
Comment (JuliaGalabut) [inline oap-hadoop/pom.xml:72]: can you, please, move it to property

File: pom.xml [hunk 3]
```diff
@@ -25,6 +25,15 @@
     </modules>
 
     <repositories>
+        <repository>
+            <id>central</id>
+            <name>Maven Central</name>
+            <layout>default</layout>
+            <url>https://repo1.maven.org/maven2</url>
```
Comment (JuliaGalabut) [inline pom.xml:32]: do we need exactly repo1?

File: oap-hadoop/pom.xml (hunk 2 above)
Comment (alebastrov) [reply oap-hadoop/pom.xml]: done

File: oap-hadoop/pom.xml (hunk 1 above)
Comment (alebastrov) [reply oap-hadoop/pom.xml]: done

File: pom.xml (hunk 3 above)
Comment (alebastrov) [reply pom.xml:32]: it was in example

**Review by JuliaGalabut — APPROVED** [review]

**Pull request merged**